packages/flutter/lib/src/animation/animation_override.dart

```diff
@@ -0,0 +1,36 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'curves.dart';
+
+/// A class that can be used to override the default animations of a widget.
+///
+/// Currently, this class is used by the following widgets:
+/// - [ExpansionTile]
+/// - [MaterialApp]
+/// - [PopupMenuButton]
+///
+/// All of the parameters are optional. If no parameters are specified,
```

Comment:
I think it would be useful to claim that if the duration and reverseDuration properties the corresponding animation will have been disabled.

Reply:
Do you mean adding a line that says this?

```dart
/// If [duration] and [reverseDuration] are set to [Duration.zero], the
/// corresponding animation will be disabled.
```

Reply:
Yes!